src/pages/workspace/WorkspaceSidebar.js

```diff
@@ -152,17 +177,19 @@ const WorkspaceSidebar = ({
                                 ]}
                                 onPress={openEditor}
                             >
-                                <Tooltip text={policy.name}>
-                                    <Text
-                                        numberOfLines={1}
-                                        style={[
-                                            styles.displayName,
-                                            styles.alignSelfCenter,
-                                        ]}
-                                    >
-                                        {policy.name}
-                                    </Text>
-                                </Tooltip>
+                                {policy.name && (
```

Comment:
Why would this policy not have a name?

Also do you think we should have something like `_.isEmpty(policy.name)`? I thought if we don't React Native will try to render an empty string and things break or a warning is thrown.

Reply:
> Why would this policy not have a name

Because the default value of the policy prop is `{}`. But yeah, I agree `_.isEmpty` is safer.